lib/import.php

```diff
@@ -86,7 +87,20 @@ public function master() {
 	 *
 	 * @return string|WP_Error
 	 */
-	protected function commit( $commit ) {
+	protected function commit( $commit , $commits_array) {
+		$head_commit = array();
+		$mod_files = array();
+		$update_all = true;
+		foreach ( $commits_array as $commit_obj ) {
+			if ( $commit_obj != null ) {
```

Comment:
Then we could remove this `null` check, unless there's a possibility of getting `null` from the API.